Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apt-get update && \
 
 
 #CERN HTCondor part taken from https://gitlab.cern.ch/batch-team/condorsubmit
-ARG HTCONDORCERN=0
+ARG HTCONDORCERN
```

Comment:
This gives warning:

```
 ---> Running in 608520108fd0
/bin/sh: 1: [: Illegal number:
```

Perhaps we can already implement the old proposition `-b BACKENDS=kubernetes,htcondorcern,slurmcern` i.e. we would be comparing strings and avoid troubles...

(... just a note, continuing with testing matrix...)

Reply:
Testing matrix:

| R-J-C Build | Workflow asks for K8S | Workflow asks for HTC |
| --- | --- | --- |
| (regular) | OK | ERR (it runs "ok", but it should fail, because user-wanted HTC backend is not available) |
| HTCONDORCERN=1 | OK | OK (well submitted to HTC, waiting for results) |